lightly/utils/scheduler.py

```diff
@@ -19,6 +24,9 @@ def cosine_schedule(
             Starting value.
         end_value:
             Target value.
+        period (optional):
+            The number of steps over which the cosine function completes its half-cycle.
```

Comment:
Are you sure this is only for a half cycle? Shouldn't `np.cos(2 * pi * step / period)` give a full cycle once `step == period`

Reply:
Sorry, yes you are absolutely correct. It is a full cycle as in the paper. Will Fix.